frontend/device/kindle/device.lua

```diff
@@ -1064,6 +1064,7 @@ function Kindle3:init()
         event_map = require("device/kindle/event_map_kindle4"),
     }
     self.keyboard_layout = require("device/kindle/keyboard_layout")
+    self.alt_plus_key_kernel_translated = dofile("frontend/device/kindle/k3_alt+qwer_conversion.lua")
```

Comment:
k3_alt+keys_kernel_translated.lua ?

Reply:
`k3_alt_and_top_row`

(Or something along those lines, i.e., what's modified is the top row of keys).